Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Last-Update: 2012-09-13
Forwarded: not-needed
Bug-Debian: http://bugs.debian.org/672150
Bug-Abiword: http://bugzilla.abisource.com/show_bug.cgi?id=13402
Description: backported patch (r31914) to fix clip art window
 make vbox expandable


--- a/src/af/xap/gtk/xap_UnixDlg_ClipArt.cpp
+++ b/src/af/xap/gtk/xap_UnixDlg_ClipArt.cpp
@@ -144,7 +144,7 @@
 	connectFocus(GTK_WIDGET(this->dlg), pFrame);
 
 	GtkWidget *vbox = gtk_box_new (GTK_ORIENTATION_VERTICAL, 10);
-	gtk_container_add (GTK_CONTAINER (gtk_dialog_get_content_area(GTK_DIALOG(this->dlg))), vbox);
+	gtk_box_pack_start(GTK_BOX (gtk_dialog_get_content_area(GTK_DIALOG(this->dlg))), vbox, TRUE,TRUE,0);
 	
 	pSS->getValueUTF8(XAP_STRING_ID_DLG_CLIPART_Loading, s);
 	this->progress = gtk_progress_bar_new ();
